Header header
float64 x
float64 xDot
float64 theta
float64 thetaDot
bool rightLim
bool leftLim
